em-app/examples/get-certificate/Cargo.toml

```diff
@@ -11,6 +9,3 @@ license = "MPL-2.0"
 em-app = { path = "../../" }
 mbedtls = { version = "0.8.1", features = [ "rdrand" ] }
```

Comment:
Can you use the mpi_force_c_code flag here as well? It's not strictly required here as em-app will already enable the feature, but you're generating a public key in this code.